PocketCastsTests/Tests/Whats New/WhatsNewTests.swift

```diff
@@ -0,0 +1,62 @@
+import XCTest
+
+@testable import podcasts
+
+class WhatsNewtests: XCTestCase {
+    /// When upgrading from 7.39 to 7.40 and there's a "What's New"
+    /// for 7.40, show it
+    func testShowWhatsNew() {
+        let whatsNew = WhatsNew(
+            announcements: [.init(version: 7.40, image: "", title: "", message: "")],
+            previousOpenedVersion: 7.39,
+            currentVersion: 7.40
+        )
+
+        XCTAssertNotNil(whatsNew.viewControllerToShow())
+    }
+
+    /// When just opening the same version, do nothing
```

Comment:
I mentioned this below, but if we add a test where all 3 versions are the same, `viewControllerToShow` will always return a view controller.

```
    func testDontShowWhatsNewAgain() {
        let whatsNew = WhatsNew(
            announcements: [.init(version: 7.40, image: "", title: "", message: "")],
            previousOpenedVersion: 7.40,
            currentVersion: 7.40
        )

        XCTAssertNil(whatsNew.viewControllerToShow())
    }
```

Reply:
Oh! Thanks for spotting that, fixed in c545923d4c104ccb24863db901aa6427e18b5f41